src/bin/coldsnap/main.rs

```diff
@@ -159,7 +159,7 @@ struct DownloadArgs {
     snapshot_id: String,
 
     #[argh(positional)]
-    filename: PathBuf,
+    file: PathBuf,
```

Comment:
> `file` which feels like a canonical and accurate way to say that we are accepting a path

Why not `path`?  The file is the object, the path is its address.

```suggestion
    path: PathBuf,
```

Reply:
I chose file because I thought it was better for the usage text:

```
Usage: coldsnap download <snapshot_id> <file> [--force] [--no-progress]

Download an EBS snapshot into a local file.

Options:
  --force           overwrite an existing file
  --no-progress     disable the progress bar
  --help            display usage information
```

Path would be fine, I guess, but it felt like `file` felt more canonical for command line program usage statements.